avocado/utils/astring.py

```diff
@@ -329,15 +329,16 @@ def to_text(data, encoding=ENCODING):
     it's returned unchanged.
 
     :param data: data to be transformed into text
-    :param encoding: encoding of the data (only used when decoding
-                     is necessary)
     :type data: either bytes or other data that will be returned
                 unchanged
+    :param encoding: encoding of the data (only used when decoding
+                     is necessary)
+    :param errors: how to handle encode/decode errors
```

Comment:
It's a good idea to reference this:

https://docs.python.org/3/library/codecs.html#error-handlers

Reply:
ok